trite-osprey/src/List.js

```diff
@@ -82,15 +92,27 @@ class List extends React.Component{
             placeholder="Search beer"
           />
 
-          <ul>
+          <ul onClick={this.show.bind(this)}>
```

Comment:
Shouldn't the onClick go onto each li? It seems like you want to have the popup for the individual beer show up when it is clicked. Right not it will show up when you click the whole list.